UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Webjob.UnitTests/Services/FulfilmentFileShareServiceTest.cs

```diff
@@ -25,9 +25,11 @@ public class FulfilmentFileShareServiceTest
         public bool fakeIsZipFileCreated = false;
         private bool fakeIsBatchCommitted = false;
         public string fakeExchangeSetRootPath = @"D:\\Downloads\";
-        public string fakeBatchId = "7b4cdf10-adfa-4ed6-b2fe-d1543d8b7272";
+        readonly string fakeBatchId = "7b4cdf10-adfa-4ed6-b2fe-d1543d8b7272";
```

Comment:
Add access modifier

Reply:
strange but access modifier is already present in this line.

Reply:
For me the line is missing a private/internal/protected/public modifier

Reply:
Resolved